### PR TITLE
Models for tiers in showback

### PR DESCRIPTION
**Pull request by aljesusg**
Tiers models for showback. Now a rate has many tiers for define a chargeback.

See documentation in https://github.com/miq-consumption/manageiq-consumption/wiki
See the repo https://github.com/miq-consumption/manageiq-consumption

@sergio-ocon I need your approval to remove WIP tag

### CODE REVIEW

**Comment by aljesusg** [comment]
@miq-bot remove_label wip

File: db/migrate/20170914082040_showback_tiers.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    remove_column    :showback_rates, :variable_rate_currency
+    remove_column    :showback_rates, :variable_rate_per_unit
+    remove_column    :showback_rates, :variable_rate_per_time
+
+    remove_column    :showback_rates, :step_value
+    remove_column    :showback_rates, :step_time_unit
+    remove_column    :showback_rates, :step_time_value
+    remove_column    :showback_rates, :step_unit
+
+    add_column       :showback_rates, :used_single_tier,       :boolean
```
Comment (sergio-ocon) [inline db/migrate/20170914082040_showback_tiers.rb]: uses instead of used?

File: db/migrate/20170914082040_showback_tiers.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    add_column       :showback_rates, :tiers_with_full_value,  :boolean
```
Comment (sergio-ocon) [inline db/migrate/20170914082040_showback_tiers.rb]: tiers_use_full_value

File: db/migrate/20170914082040_showback_tiers.rb [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+    rename_column    :showback_rates, :fixed_rate_per_time, :variable
```
Comment (sergio-ocon) [inline db/migrate/20170914082040_showback_tiers.rb]: step_variable

**Comment by sergio-ocon** [comment]
@miq-bot assign Fryguy

**Pull request merged**